web_src/js/components/DashboardRepoList.vue

```diff
@@ -475,6 +491,16 @@ export default sfc; // activate the IDE's Vue plugin
           <svg-icon name="octicon-plus"/>
         </a>
       </h4>
+      <div v-if="isLoading" class="ui attached segment" :class="{'is-loading': isLoading}"/>
+      <div v-if="!isLoading && !organizations.length" class="ui attached segment empty-placeholder">
```

Comment:
It doesn't seem right. How to distinguish the two cases?

1. the list is really empty.
2. the list is empty but it is caused by the search filter.

Reply:
Thanks for reminding me, I'll continue to refine this logic.